py3status/modules/taskwarrior.py

```diff
@@ -38,7 +41,7 @@ def taskWarrior(self):
         def describeTask(taskObj):
             return str(taskObj['id']) + ' ' + taskObj['description']
 
-        task_command = 'task start.before:tomorrow status:pending export'
+        task_command = 'task ' + self.task_args + ' export'
```

Comment:
(Nitpick) Can we use this instead?
```
        task_command = 'task %s export' % self.task_args \
            if self.task_args else 'task export'
```
The `task_command` works fine here. This sends out `task  export` (double space) if the `task_args` is `''`. We can make it a bit nicer by sending out `task export` with a single space instead. And we could finalize the `self.cmd`  once in the `post_config_hook` for the command too.

Reply:
if `task_command = 'task %s export' % self.task_args` gives the same output with the double space, just use that.

imho `\` line continuation is a bug in python

Reply:
Ideally, we should start off with `filter = None` because we expose this now. We can do this right the first time by removing the hardcoded filter and toss in a good example too.
```
Examples:
# show pending tasks
task {
    filter = 'status:pending'
}
```
If we use `task export` without `filter` above, we will get old tasks too. Maybe a limitation on their part, but this gives us more points (or data) for py3status to play with. Defaulting `filter = status:pending` is somewhat like using `\?not_zero` as the completed/deleted tasks will have `0` for `{id}`.

I'm not sure what we can do with old tasks. Maybe a count along with hiding the completed/deleted tasks to display a full count of tasks, not just `pending`.

Reply:
I'd rather keep the current default in order to ensure backward-compatibility.
Otherwise the format string is much nicer to read, and indeed guarding against `filter = None` is a sensible idea (I'm not a pythonist at all, so every comment even for nitpicking is welcome ;) )